pkg/acsengine/azureconst.go

```diff
@@ -47,6 +47,15 @@ var AzureLocations = []string{
 	"francecentral",
 }
 
+// GetDCOSBootstrapAllowedSizes returns the bootstrap allowed sizes
```

Comment:
This file is auto-generated, so these changes need to derive from `pkg/acsengine/Get-AzureConstants.py` (and then an execution of that script).